swap_meet/vendor.py

```diff
@@ -1,2 +1,196 @@
 class Vendor:
-    pass
\ No newline at end of file
+    """
+    A class to represent a Vendor.
+    
+    '''
+    Attributes
+    ----------
+    inventory (list): List of items, default is a empty list.
+
+    Methods
+    -------
+    add(item):
+        Adds an item to the inventory.
+
+    remove(item):
+        Removes an item from the inventory.
+        
+    get_by_category(category):
+        Returns a list of items with same category as argument.
+        
+    swap_items(other_vendor, my_item, their_item)
+        Swaps specific item (my_item) with another vendor's item (their_item).
+
+    swap_first_item(other_vendor)
+        Swaps the first item in inventory between this vendor and another vendor.
+
+    get_best_by_category(category)
+        Returns the item with the highest condition value in that specific category.
+
+    swap_best_by_category(other, my_priority, their_priority)
+        Swaps the items with the highest condition value in the 
+        preferred category with another vendor.
+    """
+
+    def __init__(self, inventory=None):
+        """
+        Constructs all the necessary attributes for the Vendor object.
+
+        Parameters
+        ----------
+        inventory (list, optional): List of items, default is a empty list.
+        """
+
+        self.inventory = inventory if inventory else []
+
+
+    def add(self, item):
+        """
+        Adds an item to the inventory.
+        
+        Parameters
+        ----------
+        item (str): an item to add
+
+        Returns
+        -------
+        item
+        """
+
+        self.inventory.append(item)
+        return item
+
+
+    def remove(self, item):
+        """
+        Removes an item from the inventory.
+        
+        Parameters
+        ----------
+        item(str): an item to remove
+
+        Returns
+        -------
+        Returns item if successful, False otherwise.
+        """
+
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
👍 This is a neat solution! We can also reverse the conditional into a guard clause so that the primary logic isn't indented.

```python
    def remove(self, item):
        """
        Removes an item from the inventory.
        
        Parameters
        ----------
        item(str): an item to remove

        Returns
        -------
        Returns item if successful, False otherwise.
        """

        if item not in self.inventory:
            return False
            
        self.inventory.remove(item)
        return item
```